bam-cmd/src/main/java/org/victorchang/QnameCommand.java

```diff
@@ -129,14 +133,34 @@ private boolean checkDirectory(Path path) {
     }
 }
 
+class QnameParam {
+    @Option(names = {"-n", "--name"}, description = "QNAME to search for")
+    String qname;
+    @Option(names = {"-f", "--file-name"}, description = "Path to file containing QNAMEs to search for (separated by newline).")
+    Path qnamePath;
+
+    List<String> getQnames() {
+        if (qname != null) {
+            return singletonList(qname);
+        }
+        try {
+            return Files.readAllLines(qnamePath);
```

Comment:
Let's add something like `.filter(s -> !s.isEmpty())`, because if the file contains extra newlines in the end, we don't want to search for empty strings

Reply:
Filtering out all invalid qnames from the file now.